client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -26,7 +26,14 @@ export default React.createClass( {
 		icon: React.PropTypes.string,
 		event: React.PropTypes.string,
 		href: React.PropTypes.string,
-		jetpack: React.PropTypes.bool
+		jetpack: React.PropTypes.bool,
+		feature: React.PropTypes.oneOf( [ 
+			false,
```

Comment:
I think we should this enum somewhere in const and we generally should have plans-related utils.
What do you think?